core/src/providers/tiktoken/tiktoken.rs

```diff
@@ -107,6 +107,32 @@ pub fn cl100k_base() -> Result<CoreBPE> {
     )
 }
 
+pub fn o200k_base() -> Result<CoreBPE> {
+    let o200k_base = include_str!("o200k_base.tiktoken");
+
+    let mut encoder = HashMap::default();
+    for line in o200k_base.lines() {
+        let mut parts = line.split(' ');
+        let raw = parts.next().unwrap();
+        let token = &general_purpose::STANDARD.decode(raw)?;
+        let rank: usize = parts.next().unwrap().parse().unwrap();
+        encoder.insert(token.clone(), rank);
+    }
+
+    let mut special_tokens = HashMap::default();
```

Comment:
This is also valid for this new version ?

Reply:
I have no idea I copy pasted same as other tokenizer 😬

Reply:
@spolu where did you source those originally ?

Reply:
From the tiktoken code we need to get the updated ones indeed :+1: 

I thiiink it might be on the python side of things?

Reply:
I added a commit, from this: https://github.com/openai/tiktoken/blob/main/tiktoken_ext/openai_public.py#L91C1-L92C1

(`@q` handled the migration from python to rust)